java/code/src/com/redhat/rhn/manager/system/test/SystemManagerTest.java

```diff
@@ -2088,4 +2088,17 @@ private Set<Long> listDupesByIpAddress(String byIpAdress) {
                 .map(NetworkDto::getId)
                 .collect(Collectors.toSet());
     }
+
+    public void testCountOutdatedSystems() throws Exception {
+        User user = UserTestUtils.findNewUser("testUser",
+                "testOrg" + this.getClass().getSimpleName());
+        Server server = ServerFactoryTest.createTestServer(user);
+        Long sid = server.getId();
+        Package pack = PackageTest.createTestPackage(user.getOrg());
+
+        ErrataCacheManager.insertNeededErrataCache(sid, null, pack.getId());
+        SystemsOverviewUpdateWorker.doUpdate(sid);
```

Comment:
This updates the new `suseSystemsOverview` table which you are not using in this PR. This would be needed though with my suggested changes